docs/documentation/how-tos/awareness-information.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Awareness Information
+---
+
+## Follow-Mode
```

Comment:
:+1: Seems like a good overview of the available awareness information. How about adding a link to this page to the "Awareness Information" section on the landing page?

It should be mentioned that the "Follow-Mode", "File Navigation", and "Viewports" sections currently only apply to Saros/E as these features have not yet been implemented for Saros/I.